ipq40xx: convert GL.iNet GL-S1300 to DSA

PolynomialDivision opened the pull request:
Convert the router to DSA.

mattmatician commented:
@PolynomialDivision Thanks for this PR. I have a GL.iNet GL-S1300, is this something I can help by testing?

PolynomialDivision committed:
ipq40xx: convert GL.iNet GL-S1300 to DSA

Convert the router to DSA.

Co-Developed-by: Matt Beaumont <github@beaum.xyz>
Tested-by: Matt Beaumont <github@beaum.xyz>
Link: https://github.com/openwrt/openwrt/pull/12478
Signed-off-by: Nick Hainke <vincent@systemli.org>

PolynomialDivision commented:
@mattmatician I updatet the PR. It would be nice if you could test it. However, you may lose ethernet connectivity, so you should build an image with wifi enabled by default.

PolynomialDivision commented:
Thanks! Rebased on top of main and merged!
